alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/ui/ChannelDistributionUIConfig.java

```diff
@@ -50,48 +46,61 @@
 
     private static final String LABEL_NAME = "Name";
     private static final String LABEL_FREQUENCY = "Frequency";
-    private static final String LABEL_CHANNEL_NAME = "Channel Type";
+    private static final String LABEL_CHANNEL_NAME = "Type";
     private static final String LABEL_PROVIDER_NAME = "Provider Type";
 
     private static final String DESCRIPTION_NAME = "The name of the distribution job. Must be unique.";
     private static final String DESCRIPTION_FREQUENCY = "Select how frequently this job should check for notifications to send.";
     private static final String DESCRIPTION_CHANNEL_NAME = "Select the channel. Notifications generated through Alert will be sent through this channel.";
     private static final String DESCRIPTION_PROVIDER_NAME = "Select the provider. Only notifications for that provider will be processed in this distribution job.";
 
-    private final Logger logger = LoggerFactory.getLogger(ChannelDistributionUIConfig.class);
-    private final DescriptorAccessor descriptorAccessor;
+    private final DescriptorMap descriptorMap;
```

Comment:
I thought we were trying to get rid of DescriptorMap?

Reply:
We were trying to get rid of descriptorMap and replace it with the DB calls, but for the auto-generated UI we actually needed a bit more data from the descriptor such as name, label and fontawesome icon. If we stored this data in the DB, then that'd be it. But unfortunately we do and I don't think that we should.